docker-compose.yml

```diff
@@ -31,10 +39,22 @@ services:
       - ./configuration/postgresql/initdb:/docker-entrypoint-initdb.d
       - postgresqldata:/var/lib/postgresql/data
 
+  postgresCsV4:
+    image: postgres:bookworm
+    volumes:
+      - ./configuration/postgresCsV4/initdb:/docker-entrypoint-initdb.d
+      - postgresqldata:/var/lib/postgresCsV4/data
+    environment:
+      - POSTGRES_PASSWORD=root
+    ports: 
+      - 5435:5432
+      
   redis:
     image: redis:5-alpine
     volumes:
       - redisdata:/data
+    ports: 
+      - 6379:6379
```

Comment:
Technically not needed, since these are the default ports

Reply:
Im pretty sure I had issues getting the redis from the migration script if this was not set. I think docker will open the port with this (but not sure).